doc/CMakeLists.txt

```diff
@@ -6,7 +6,14 @@ if (NOT DOXYGEN)
   return ()
 endif ()
 
-find_package(PythonInterp QUIET REQUIRED)
+# Find the python interpreter, set the PYTHON_EXECUTABLE variable
```

Comment:
nit: Please use proper punctuation in comments for consistency with the rest of the library.